tests/test_home.py

```diff
@@ -105,12 +109,14 @@ def test_mozilla_footer_link(base_url, selenium):
     ),
 )
 @pytest.mark.nondestructive
-def test_addons_footer_links(base_url, selenium, count, links):
-    page = Home(selenium, base_url).open()
+def test_addons_footer_links(base_url, selenium, page, count, links):
+    selenium.get(f'{base_url}{page}')
+    page = Base(selenium, base_url)
     page.footer.addon_links[count].click()
     page.wait_for_current_url(links)
 
 
+@pytest.mark.parametrize('page', ['', '/developers'])
```

Comment:
I find explicit IDs for test parameters very useful for producing more comprehensible test reports. I blogged about this a few years ago. The described APIs should still work. Hope this helps! 📝 

https://raphael.codes/blog/ids-for-pytest-fixtures-and-parametrize/